pkgs/development/ocaml-modules/earlybird/default.nix

```diff
@@ -1,30 +1,27 @@
 { lib, fetchFromGitHub, ocaml, buildDunePackage
 , cmdliner, dap, fmt, iter, logs, lru, lwt_ppx, lwt_react, menhir, menhirLib, path_glob, ppx_deriving_yojson
+, ppx_optcomp
 , gitUpdater
 }:
 
-if lib.versionAtLeast ocaml.version "4.13"
-then throw "earlybird is not available for OCaml ${ocaml.version}"
-else
-
 buildDunePackage rec {
   pname = "earlybird";
-  version = "1.1.0";
-
-  duneVersion = "3";
+  version = "1.2.1";
 
-  minimalOCamlVersion = "4.11";
+  minimalOCamlVersion = "4.12";
 
   src = fetchFromGitHub {
     owner = "hackwaly";
     repo = "ocamlearlybird";
     rev = version;
-    hash = "sha256-8JHZWsgpz2pzpDxST3bkMSmPHtj7MDzD5G3ujqMW+MU=";
+    hash = "sha256-p29uTdx8+mZKXUL+ng/FzpKuhnykEe8Sy968Wa/KUn4=";
   };
 
   nativeBuildInputs = [ menhir ];
 
-  buildInputs = [ cmdliner dap fmt iter logs lru lwt_ppx lwt_react menhirLib path_glob ppx_deriving_yojson ];
+  buildInputs = [ cmdliner dap fmt iter logs lru lwt_ppx lwt_react menhirLib path_glob
+    ppx_deriving_yojson ppx_optcomp
```

Comment:
This is some wild spacing. Any reason to not just newline per input?

Reply:
Fixed.